moped-editor/src/views/projects/projectView/ProjectFunding/LookupAutocompleteComponent.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Autocomplete, TextField } from "@mui/material";
+import { useGridApiContext } from "@mui/x-data-grid-pro";
+import makeStyles from "@mui/styles/makeStyles";
+import { getLookupValueByID } from "./utils/helpers";
+import CustomPopper from "src/components/CustomPopper";
+
+const useStyles = makeStyles((theme) => ({
+  autocompleteLookupInput: {
+    minWidth: "200px",
+    alignContent: "center",
+    padding: theme.spacing(1),
+  },
+}));
+
+/**
+ * Component for dropdown select using a lookup table as options
+ * @param {*} props
+ * @returns {React component}
+ */
+const LookupAutocompleteComponent = (props) => {
```

Comment:
slick component! it'd be nice to have the props destructured just so it's easy to see what need to get passed to this function

Reply:
destructured!